### PR TITLE
Create movie-quote

### PR DESCRIPTION
**Pull request by therefat**

### CODE REVIEW

File: app/messages/movie-quote [hunk 1]
```diff
@@ -0,0 +1,2 @@
+By: <The Shawshank Redemption (1994)>
```
Comment (SavvasStephanides) [inline app/messages/movie-quote:1]: Thanks, Refat! Can you please update your file so that the `By:` line contains your own name?

> By: Refat Hossen

**Review by SavvasStephanides — CHANGES_REQUESTED** [review]
Thank you for your submission @therefat! Some suggestions to update your file befoe approving. 🙂

File: app/messages/movie-quote (hunk 1 above)
Comment (therefat) [reply app/messages/movie-quote]: @SavvasStephanides  please check again. now update done

**Review by therefat — COMMENTED** [review]
update done

**Comment by SavvasStephanides** [comment]
Fantastic! If you want, you can also mention "The Shawshank Redemption" at the end of your message:

> Remember Red, hope is a good thing, maybe the best of things, and no good thing ever dies. -- The Shawshank Redemption (1994)

**Review by SavvasStephanides — APPROVED** [review]

**Review by therefat — COMMENTED** [review]
done check again

**Comment by SavvasStephanides** [comment]
Your message is up on the site: https://just-say-something.vercel.app/message/movie-quote

Thank you for your submission, @therefat!

**Pull request merged**